### PR TITLE
Application monitoring operator v1.6.1

### PR DESCRIPTION
**Pull request by jackdelahunt**
# Description
This pr updates the application monitoring operator we use from `v1.6.0` to `v1.6.1`.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Verify
- AMO CSV is for v1.6.1 as shown below.
image
- The Application monitoring containers are using the 1.6.1 image. 
image
- Go to Middleware Monitoring > Routes > Grafana and verify all expected dashboards are present and are fucntional.

### CODE REVIEW

File: manifests/integreatly-monitoring/1.6.1/application-monitoring-operator.v1.6.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+  - Grafana
+  - Monitoring
+  - AlertManager
+  maintainers:
+  - email: placeholder@email.com
+    name: Integreatly
+  maturity: alpha
+  provider:
+    name: Red Hat
+  replaces: application-monitoring-operator.v1.5.1
```
Comment (KevFan) [inline manifests/integreatly-monitoring/1.6.1/application-monitoring-operator.v1.6.1.clusterserviceversion.yaml:353]:
```suggestion
  replaces: application-monitoring-operator.v1.6.0
```
Should this be replaces `1.6.0` ? :thinking:

Comment (jackdelahunt) [reply manifests/integreatly-monitoring/1.6.1/application-monitoring-operator.v1.6.1.clusterserviceversion.yaml:353]: Yep you're right.

**Comment by KevFan** [comment]
* Verfied fresh install on this PR
  * Verified AMO csv version is `1.6.1`
  * Verified AMO operator image is `quay.io/integreatly/application-monitoring-operator:1.6.1`
  * Verified grafana operater image in middlware monitoring namespace is `quay.io/integreatly/grafana-operator:v3.10.2`
  * Verified dashboards are present and functional
  
Screenshot from 2021-07-02 12-30-31

* Verfied upgrade from master to this PR
  * Verified AMO csv version is upgraded from `1.6.0` to  `1.6.1`
  * Verified AMO operator image is upgraded from  `quay.io/integreatly/application-monitoring-operator:1.6.0` to `quay.io/integreatly/application-monitoring-operator:1.6.1`
  * Verified grafana operater image in middlware monitoring namespace is upgraded from `quay.io/integreatly/grafana-operator:v3.10.1` to `quay.io/integreatly/grafana-operator:v3.10.2`
  * Verified dashboards are present and functional

Screenshot from 2021-07-02 14-03-59
Screenshot from 2021-07-02 14-09-09

Nice work :+1: 

/lgtm

**Comment by philbrookes** [comment]
/approve
ping @KevFan can we get a fresh lgtm on this please?

**Comment by KevFan** [comment]
Re-verified install and upgrade of AMO following rebase :+1: 

/lgtm

**Pull request merged**